.ci/setup_env.sh

```diff
@@ -1,112 +1,34 @@
 #!/usr/bin/env bash
-set -e
+# set -e
```

Comment:
did you mean to nuke that?

Reply:
Yes. This conflicts with the Travis exit handler and causes builds to fail without any error message shown. Which was a big pain while I was troubleshooting.